pkg/asset/manifests/tectonic.go

```diff
@@ -89,3 +94,12 @@ func (t *Tectonic) Generate(dependencies asset.Parents) error {
 func (t *Tectonic) Files() []*asset.File {
 	return t.files
 }
+
+// Load returns the tectonic asset from disk.
+func (t *Tectonic) Load(p asset.FilePatternFetcher) (found bool, err error) {
+	files, found, err := p(fmt.Sprintf("%s/*", tectonicManifestDir))
```

Comment:
This is going to load the config file from the Manifests asset, too, right?

Reply:
Same as the `manifest/*` case, we will load everything under `tectonic/`.

Reply:
It will read the "00_cluster-config.yaml" file that was created by the Manifests asset, not this asset.